USER_GUIDE.md

```diff
@@ -0,0 +1,86 @@
+# User guide
+
+---
+Primitive guide for users 1.3.0 and above (need to expand)
+
+# App Engine configuration file deployment
+
+You can now deploy the cron/doc/etc. configuration files separately using the new goals:
+
+* `deployCron`
+* `deployDispatch`
+* `deployDos`
+* `deployIndex`
+* `deployQueue`
+
+## Source directory
+_For GAE Flexible projects_ The deployment source directory can be overridden by setting the `appEngineDirectory` parameter in the deploy configuration.
```

Comment:
Oh right, never mind that makes sense. I was thinking "For _GAE Flexible projects_", but I think what you have makes more sense.